lms/djangoapps/student_account/views.py

```diff
@@ -41,6 +41,9 @@
 
 from openedx.core.djangoapps.user_api.accounts.api import request_password_change
 from openedx.core.djangoapps.user_api.errors import UserNotFound
+
+import pyuca
```

Comment:
This needs to be moved up: https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order